ui/page/wallets/restore_page.go

```diff
@@ -401,20 +405,40 @@ func (pg *Restore) updateSeedResetBtn() bool {
 	return false
 }
 
-func (pg *Restore) validateSeeds() bool {
-	pg.seedPhrase = ""
+func (pg *Restore) validateSeeds() (bool, string) {
+	focus := pg.seedEditors.focusIndex
+	seedMatchCounter := 0
+	seedPhraseStore := ""
```

Comment:
kindly comment this so we know what it does... not sure what `seedPhraseStore` is and can we use a better variable name?